@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fn_getVelocities.sqf

```diff
@@ -0,0 +1,54 @@
+/* ----------------------------------------------------------------------------
+Function: fza_sfmplus_fnc_rotorMain
+
+Description:
+    Simple rotor provides a simple, grounded in reality simulation of a
+    helicopters rotor. Translational Lift, Ground Effect and Vortex Ring State
+    are all simulated. This is an implementation of NASA Technical Paper 1285
+    bu J.D. Shaughnessy with simplifications sourced from JSBSim.
+
+Parameters:
+    _heli - The helicopter to get information from [Unit].
+
+Returns:
+    _gndSpeed, kts
+    _vel2D, kts, simulates velocity from pitot/static sources
+    _vel3D, kts, simulates velocity from an air data sensor
+    _velModelSpace, m/s, unified local velocity for the simulation to use
+    _velWorldSpace, m/s, unified velocity for the simulation to use
+
+Examples:
+    ...
+
+Author:
+    BradMick
+---------------------------------------------------------------------------- */
+params ["_heli", "_useWind"];
+
+#include "\fza_ah64_sfmplus\headers\core.hpp"
+
+private _gndSpeed = round(MPS_TO_KNOTS * (velocityModelSpace _heli select 1));
+
+private _vel3D         = 0.0;
+private _vel2D         = 0.0;
+private _velModelSpace = [];
+private _velWorldSpace = [];
+
+if (_useWind) then {
+    _vel3D         = round(MPS_TO_KNOTS * vectorMagnitude(velocityModelSpace _heli vectorDiff wind));
```

Comment:
Couldn't these be the same, but instead of calling `wind` directly, you just have a vector which you always diff, either with the contents of `wind` or `[0,0,0]` That will reduce the number of lines and reduce the chance of the two becoming misaligned